src/constants/constants.js

```diff
@@ -1,2 +1,14 @@
+export const DEFAULT_SVG_STRING = '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64" width="64" height="64"><radialGradient id="IkUh6ey5BhnCh~hrMs1fda" cx="32.5" cy="31.5" r="30.516" gradientUnits="userSpaceOnUse" spreadMethod="reflect"><stop offset="0" stop-color="#afeeff"/><stop offset=".193" stop-color="#bbf1ff"/><stop offset=".703" stop-color="#d7f8ff"/><stop offset="1" stop-color="#e1faff"/></radialGradient><path fill="url(#IkUh6ey5BhnCh~hrMs1fda)" d="M59,20h1.5c2.168,0,3.892-1.998,3.422-4.243C63.58,14.122,62.056,13,60.385,13L53,13 c-1.105,0-2-0.895-2-2c0-1.105,0.895-2,2-2h3.385c1.67,0,3.195-1.122,3.537-2.757C60.392,3.998,58.668,2,56.5,2H34.006H32.5h-24 C6.575,2,5,3.575,5,5.5S6.575,9,8.5,9H10c1.105,0,2,0.895,2,2c0,1.105-0.895,2-2,2l-5.385,0c-1.67,0-3.195,1.122-3.537,2.757 C0.608,18.002,2.332,20,4.5,20H18v12L4.615,32c-1.67,0-3.195,1.122-3.537,2.757C0.608,37.002,2.332,39,4.5,39H5c1.105,0,2,0.895,2,2 c0,1.105-0.895,2-2,2H4.5c-2.168,0-3.892,1.998-3.422,4.243C1.42,48.878,2.945,50,4.615,50H10c1.105,0,2,0.895,2,2 c0,1.105-0.895,2-2,2l-1.385,0c-1.67,0-3.195,1.122-3.537,2.757C4.608,59.002,6.332,61,8.5,61h22.494H32.5h23 c1.925,0,3.5-1.575,3.5-3.5S57.425,54,55.5,54H55c-1.105,0-2-0.895-2-2c0-1.105,0.895-2,2-2h4.385c1.67,0,3.195-1.122,3.537-2.757 C63.392,44.998,61.668,43,59.5,43H47V31h12.385c1.67,0,3.195-1.122,3.537-2.757C63.392,25.998,61.668,24,59.5,24H59 c-1.105,0-2-0.895-2-2C57,20.895,57.895,20,59,20z"/><linearGradient id="IkUh6ey5BhnCh~hrMs1fdb" x1="32" x2="32" y1="56" y2="6" gradientUnits="userSpaceOnUse" spreadMethod="reflect"><stop offset="0" stop-color="#ff634d"/><stop offset=".204" stop-color="#fe6464"/><stop offset=".521" stop-color="#fc6581"/><stop offset=".794" stop-color="#fa6694"/><stop offset=".989" stop-color="#fa669a"/><stop offset="1" stop-color="#fa669a"/></linearGradient><path fill="url(#IkUh6ey5BhnCh~hrMs1fdb)" d="M57,31c0,13.805-11.195,25-25,25S7,44.805,7,31S18.195,6,32,6S57,17.195,57,31z"/><path fill="#fff" d="M43.268,19.732L43.268,19.732c0.976,0.976,0.976,2.559,0,3.535L24.267,42.268 c-0.976,0.976-2.559,0.976-3.535,0l0,0c-0.976-0.976-0.976-2.559,0-3.535l19.001-19.001C40.709,18.756,42.292,18.756,43.268,19.732 z"/><path fill="#fff" d="M43.268,42.268L43.268,42.268c-0.976,0.976-2.559,0.976-3.535,0L20.732,23.267 c-0.976-0.976-0.976-2.559,0-3.535l0,0c0.976-0.976,2.559-0.976,3.535,0l19.001,19.001C44.244,39.709,44.244,41.292,43.268,42.268z"/></svg>'
```

Comment:
this looks like test data, is it meant to be in this PR

Reply:
This will be an svg that wil display just incase something happens and svg cant be displayed.

Reply:
can you attach what that image is to this comment

Reply:
[data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA2NCA2NCIgd2lkdGg9IjY0IiBoZWlnaHQ9IjY0Ij48cmFkaWFsR3JhZGllbnQgaWQ9IklrVWg2ZXk1QmhuQ2h+aHJNczFmZGEiIGN4PSIzMi41IiBjeT0iMzEuNSIgcj0iMzAuNTE2IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIgc3ByZWFkTWV0aG9kPSJyZWZsZWN0Ij48c3RvcCBvZmZzZXQ9IjAiIHN0b3AtY29sb3I9IiNhZmVlZmYiLz48c3RvcCBvZmZzZXQ9Ii4xOTMiIHN0b3AtY29sb3I9IiNiYmYxZmYiLz48c3RvcCBvZmZzZXQ9Ii43MDMiIHN0b3AtY29sb3I9IiNkN2Y4ZmYiLz48c3RvcCBvZmZzZXQ9IjEiIHN0b3AtY29sb3I9IiNlMWZhZmYiLz48L3JhZGlhbEdyYWRpZW50PjxwYXRoIGZpbGw9InVybCgjSWtVaDZleTVCaG5DaH5ock1zMWZkYSkiIGQ9Ik01OSwyMGgxLjVjMi4xNjgsMCwzLjg5Mi0xLjk5OCwzLjQyMi00LjI0M0M2My41OCwxNC4xMjIsNjIuMDU2LDEzLDYwLjM4NSwxM0w1MywxMyBjLTEuMTA1LDAtMi0wLjg5NS0yLTJjMC0xLjEwNSwwLjg5NS0yLDItMmgzLjM4NWMxLjY3LDAsMy4xOTUtMS4xMjIsMy41MzctMi43NTdDNjAuMzkyLDMuOTk4LDU4LjY2OCwyLDU2LjUsMkgzNC4wMDZIMzIuNWgtMjQgQzYuNTc1LDIsNSwzLjU3NSw1LDUuNVM2LjU3NSw5LDguNSw5SDEwYzEuMTA1LDAsMiwwLjg5NSwyLDJjMCwxLjEwNS0wLjg5NSwyLTIsMmwtNS4zODUsMGMtMS42NywwLTMuMTk1LDEuMTIyLTMuNTM3LDIuNzU3IEMwLjYwOCwxOC4wMDIsMi4zMzIsMjAsNC41LDIwSDE4djEyTDQuNjE1LDMyYy0xLjY3LDAtMy4xOTUsMS4xMjItMy41MzcsMi43NTdDMC42MDgsMzcuMDAyLDIuMzMyLDM5LDQuNSwzOUg1YzEuMTA1LDAsMiwwLjg5NSwyLDIgYzAsMS4xMDUtMC44OTUsMi0yLDJINC41Yy0yLjE2OCwwLTMuODkyLDEuOTk4LTMuNDIyLDQuMjQzQzEuNDIsNDguODc4LDIuOTQ1LDUwLDQuNjE1LDUwSDEwYzEuMTA1LDAsMiwwLjg5NSwyLDIgYzAsMS4xMDUtMC44OTUsMi0yLDJsLTEuMzg1LDBjLTEuNjcsMC0zLjE5NSwxLjEyMi0zLjUzNywyLjc1N0M0LjYwOCw1OS4wMDIsNi4zMzIsNjEsOC41LDYxaDIyLjQ5NEgzMi41aDIzIGMxLjkyNSwwLDMuNS0xLjU3NSwzLjUtMy41UzU3LjQyNSw1NCw1NS41LDU0SDU1Yy0xLjEwNSwwLTItMC44OTUtMi0yYzAtMS4xMDUsMC44OTUtMiwyLTJoNC4zODVjMS42NywwLDMuMTk1LTEuMTIyLDMuNTM3LTIuNzU3IEM2My4zOTIsNDQuOTk4LDYxLjY2OCw0Myw1OS41LDQzSDQ3VjMxaDEyLjM4NWMxLjY3LDAsMy4xOTUtMS4xMjIsMy41MzctMi43NTdDNjMuMzkyLDI1Ljk5OCw2MS42NjgsMjQsNTkuNSwyNEg1OSBjLTEuMTA1LDAtMi0wLjg5NS0yLTJDNTcsMjAuODk1LDU3Ljg5NSwyMCw1OSwyMHoiLz48bGluZWFyR3JhZGllbnQgaWQ9IklrVWg2ZXk1QmhuQ2h+aHJNczFmZGIiIHgxPSIzMiIgeDI9IjMyIiB5MT0iNTYiIHkyPSI2IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIgc3ByZWFkTWV0aG9kPSJyZWZsZWN0Ij48c3RvcCBvZmZzZXQ9IjAiIHN0b3AtY29sb3I9IiNmZjYzNGQiLz48c3RvcCBvZmZzZXQ9Ii4yMDQiIHN0b3AtY29sb3I9IiNmZTY0NjQiLz48c3RvcCBvZmZzZXQ9Ii41MjEiIHN0b3AtY29sb3I9IiNmYzY1ODEiLz48c3RvcCBvZmZzZXQ9Ii43OTQiIHN0b3AtY29sb3I9IiNmYTY2OTQiLz48c3RvcCBvZmZzZXQ9Ii45ODkiIHN0b3AtY29sb3I9IiNmYTY2OWEiLz48c3RvcCBvZmZzZXQ9IjEiIHN0b3AtY29sb3I9IiNmYTY2OWEiLz48L2xpbmVhckdyYWRpZW50PjxwYXRoIGZpbGw9InVybCgjSWtVaDZleTVCaG5DaH5ock1zMWZkYikiIGQ9Ik01NywzMWMwLDEzLjgwNS0xMS4xOTUsMjUtMjUsMjVTNyw0NC44MDUsNywzMVMxOC4xOTUsNiwzMiw2UzU3LDE3LjE5NSw1NywzMXoiLz48cGF0aCBmaWxsPSIjZmZmIiBkPSJNNDMuMjY4LDE5LjczMkw0My4yNjgsMTkuNzMyYzAuOTc2LDAuOTc2LDAuOTc2LDIuNTU5LDAsMy41MzVMMjQuMjY3LDQyLjI2OCBjLTAuOTc2LDAuOTc2LTIuNTU5LDAuOTc2LTMuNTM1LDBsMCwwYy0wLjk3Ni0wLjk3Ni0wLjk3Ni0yLjU1OSwwLTMuNTM1bDE5LjAwMS0xOS4wMDFDNDAuNzA5LDE4Ljc1Niw0Mi4yOTIsMTguNzU2LDQzLjI2OCwxOS43MzIgeiIvPjxwYXRoIGZpbGw9IiNmZmYiIGQ9Ik00My4yNjgsNDIuMjY4TDQzLjI2OCw0Mi4yNjhjLTAuOTc2LDAuOTc2LTIuNTU5LDAuOTc2LTMuNTM1LDBMMjAuNzMyLDIzLjI2NyBjLTAuOTc2LTAuOTc2LTAuOTc2LTIuNTU5LDAtMy41MzVsMCwwYzAuOTc2LTAuOTc2LDIuNTU5LTAuOTc2LDMuNTM1LDBsMTkuMDAxLDE5LjAwMUM0NC4yNDQsMzkuNzA5LDQ0LjI0NCw0MS4yOTIsNDMuMjY4LDQyLjI2OHoiLz48L3N2Zz4=](url)

Reply:
I will no longer use it. I will remove those constants as well.